cypress/integration/1224-user_administration/363-administrator_opens_the_add_role_page_part_i.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Administrator opens the Add Role Dialog
+
+    @363
+    Scenario Outline: Administrator opens the Add Role Dialog
+        Given that the User is logged in as Administrator
+        And they are on the User Administration Page
+        When they click a "<Button>" under a "<Section>"

```

Comment:
`When they click a "New Administrator" under a "Administrator"`
veldig raus bruk av "a" 😅 
det er sikkert ingen automatikk i at teksten i And, They osv til cucumber oppdateres om vi oppdaterer det i scenarioene?

Reply:
Nei, da må jeg inn og endre teksten i testen